### ISSUE TITLE
[BUG] Short description of the bug

### ISSUE THREAD

**Issue by raivisdejus** [Sav22999/common-voice-android#207]
##### About OS *

_OS version:_ `Android 12`

_OS language (in English):_ `Latvian` 

##### About the app (CV Project) *

_App release:_ `2.5.0.1 (build#182::GPS)`

_App language (in **English**):_ `English`

##### About your device

_Device RAM:_ `6GB  + 2GB (Virtual RAM / swap)`

_Device display size (in Inches):_ `6.67in`

_Device model (and brand):_ `Xiaomi Poco X3 Pro`

_From what store have you installed the app?_  `Google Play Store`

_Are there installed Google Services on your device?_ `Yes`

#### **Description bug** *

Daily goal notification is not displayed in the app. I would imagine the app would show me a notification on the phone that I did not do the number of contributions I wanted. But no notification is displayed at any time. All notification permissions enabled in the app and on the phone. Xiaomi battery saver disabled for the CV project 

#### **How to reproduce the bug** *

1. Enable daily goal notification
2. Set daily goal to 50
3. Validate 10 sentences and wait for the expected notification time (with phone locked)

#### Log file

log.txt file is not created anywhere in the phone storage, at least phone search does not find any.
The `Internal storage` > `Android` >  `data` is empty
Also, I do not get the toast message when I enable the "Save logs to a file", I see from the translations there is supposed to be such a toast.

#### **Additional context**

This is a super useful app. Thanks for your time in making it. With it working with Common Voice is much more productive.

Developer mode is enabled on the phone. I am somewhat familiar with Android Studio, can do some debugging if you guide me on what to look for.

**Comment by Sav22999** [comment]
@raivisdejus The app doesn't use notifications via Google, so sometimes they don't arrive because of OS battery optimisation.

Have you set a daily goal time notifications? You can set until 2 time

**Comment by raivisdejus** [comment]
@Sav22999 Yes I did set two times when it should show up and no notification at any of the times

**Comment by Sav22999** [comment]
Mh, I don't know, I'm sorry.
